### ISSUE TITLE
JAWS fails on splash.html

### ISSUE THREAD

**Issue by psiservices-uwidmark** [FreedomScientific/standards-support#200]
### Summary

JAWS does not work if the page URL ends with splash.html# but works if it ends with splosh.html#

This was found because on a page called splash.html we had a link that opens a dialog using code on this form `<a href="#" onclick="foo()">` and when clicked this adds the # to the URL. This can be avoided by using an actual button, by changing the onclick to return false, changing the href.. anything that doesn't add the '#'

The problem can also be reproduced without the `<a href` if loading the page with # in the URL.

I am having some problem accepting that the URL matters, don't really see what JAWS would be doing with it. But it happens every time.

Steps:
1. Go to http://beta.psionline.com/jawstest/splash.html
2. Tab to the copyright link, press enter
3. Tab to the next link, hear it stop reading in the middle of the word
4. After this JAWS doesn't really function on the page, doesn't read anything, virtual cursor does not move with the arrow keys.
5. Go to http://beta.psionline.com/jawstest/splosh.html which is identical except for the page name, repeat the steps above and it works.

Alternative steps without the onclick handler:
1. Go to http://beta.psionline.com/jawstest/splash.html#
2. JAWS isn't reading much or anything
3. Go to http://beta.psionline.com/jawstest/splosh.html#
4. JAWS works fine

### Expected result
URL shouldn't matter.

### Actual result
Works on splosh.html#
Fails on splash.html#

### Example
As steps above, or see this video. I tab back and forth to test this with/without enter but you can hear JAWS read the enter or see when the # is added to the URL.
http://beta.psionline.com/jawstest/splash-splosh.mp4

### Additional Information
Reported by a customer on our actual splash.html page, and confirmed by our QA that the splash/splosh name change makes a difference. So it isn't just me and the voices in my head...

#### JAWS version and build number
2019.1904.60
 
#### Operating System and version
Windows 10, running in virtualbox.
Unsure of the other environments, but doubt virtualbox was involved.

#### Browser and version:
Chrome 74.0
FF 66.0
IE 11.765.17134.0

**Comment by ggordon-vfo** [comment]
This turns out to be the confluence of two issues. JAWS for some inexplicable reason has a config set for an app called splash  where we set JAWS to be silent when that app is in focus. It's anybody's guess why this config exists. It's been in the product since before we started using source control in 2000.

This wouldn't be an issue for the page in question were it not for the fact that putting a trailing # in the URL causes JAWS to think that splash represents the domain portion of the URL and load the config set called splash.

The quick fix will be to simply remove the config set for splash. This will happen for the JAWS July update which will release at the end of the month.
We'll also investigate why our URL parsing is failing.
